web_console/constants/index.js

```diff
@@ -1,6 +1,9 @@
 const DEFAULT_SERVER_CONFIG = {
   K8S_HOST: '127.0.0.1',
   K8S_PORT: 8000,
+  API_VERSION: 'fedlearner.k8s.io/v1alpha1',
+  KIND: 'FLApp',
+  NAMESPACE: 'default',
```

Comment:
these params would be better to be designed as the `constructor` params of `KubernetesClient`.

server config is mainly used for process config, mostly the address of remote server